tests/DCPS/DPFactoryQos/run_test.pl

```diff
@@ -12,19 +12,17 @@
 use strict;
 
 my $test = new PerlDDS::TestFramework();
-my $is_rtps_disc = 0;
 my $DCPScfg = "";
 
 if ($ARGV[0] eq "rtps_disc") {
   $DCPScfg = "-DCPSConfigFile " . $ARGV[0] . ".ini ";
```

Comment:
Now $DCPScfg is unused so this block can go away (or simply revert the file to its state before the PR started)